uiowa/uiowa
# Add additional default person types (adjunct, visiting, postdoc, alumni)

This is based off of academic site needs for CLAS and Nursing (maybe others). Resolves #3310.

Faculty already covers full faculty, associate faculty, assistant faculty, lecturer, and other instructional track faculty.
Adjunct faculty is needed to cover secondary/tertiary appointments or people who are not technically part of the primary faculty.
Visiting faculty is needed to cover temporary appointments, external to the University, often without HawkIDs.
Postdoc is needed to cover post doctoral graduates who may be performing research on campus.
Alumni is self-explanatory.

Sampada put this best, when she mentioned "Prospective students can currently see Adjunct or Visiting Faculty listed as Faculty, and are interested in their research, but by the time they get here, the faculty member is gone. Its false advertising."

# How to test

- Pull PR.
- Check that these four new Person Types exist under Admin > Structure > Person Types.
- Check that these four new Person Types exist when adding a new person.
- Check that these types appear at the bottom of an individual profile when checked.
